### DISCUSSION TITLE
[Question][Hw6] Meaning of update_seeds function

### DISCUSSION THREAD

**Discussion by anisrashidov** [prosyslab-classroom/cs348-information-security#435] [Q&A]
What was the intention behind success, mutant and new_coverage variables in  "let update_seeds env success coverage new_coverage mutant seeds = ..."?
Thanks in advance.

**Comment by DoHunLee1** [comment]
I think success, new_coverage can be used for metrics of function that is for updating seeds. If the conditions are met, mutant will add in seeds.

**Comment by m-spitfire** [comment]
FYI: you don't have to use that signature or that function, you can write your own way of updating the seeds

**Comment by KihongHeo** [comment]
Can you make a title so that other people can easily see the point?
